components/core/header/Header.vue

```diff
@@ -28,6 +25,11 @@ export default {
         NavBar,
         TextButton,
     },
+    methods: {
+        getMyPyConUrl() {
+            return `https://tw.pycon.org/2021/${this.$i18n.locale}/dashboard/`
+        },
+    },
```

Comment:
Consider using the `computed` property for consistency

Reply:
updated